lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-rc.5",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "0.0.0"
+}
```

Comment:
This is auto-generated by lerna, I wonder if it forces spacing indentation if I replace with tabs. I'll test.